Galaxus.Functional.NUnitExtension/Galaxus.Functional.NUnitExtension.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
```

Comment:
Tried to pack a release - and it didn't do anything. Well duh, didn't notice the `<IsPackable>false</IsPackable>` at all.

Also - the nuspec calls it `Galaxus.Functional.NUnit`, while the project and namespace are `Galaxus.Functional.NUnitExtensions`. I prefer the former, can you rename all `Galaxus.Functional.NUnitExtensions` to `Galaxus.Functional.NUnit`?